src/base/bittorrent/torrentinfo.cpp

```diff
@@ -276,22 +276,10 @@ TorrentInfo::PieceRange TorrentInfo::filePieces(int fileIndex) const
                         static_cast<int>((firstOffset + fileSize - 1) / pieceLength()));
 }
 
-libtorrent::file_storage TorrentInfo::files() const
-{
-    if (!isValid()) return libtorrent::file_storage();
-    return m_nativeInfo->files();
-}
-
-libtorrent::file_storage TorrentInfo::origFiles() const
-{
-    if (!isValid()) return libtorrent::file_storage();
-    return m_nativeInfo->orig_files();
-}
-
 void TorrentInfo::renameFile(uint index, const QString &newPath)
 {
     if (!isValid()) return;
-    nativeInfo()->rename_file(index, newPath.toStdString());
+    nativeInfo()->rename_file(index, Utils::Fs::toNativePath(newPath).toStdString());
```

Comment:
Apparently, I checked that, or it was in the old code. In any case, it did not cause problems, at least.